node/core/dispute-coordinator/src/real/initialized.rs

```diff
@@ -66,6 +68,9 @@ use super::{
 	OverlayedBackend,
 };
 
+const LRU_SCRAPED_BLOCKS_CAPACITY: usize = 20;
```

Comment:
This should be larger than `MAX_BATCH_SCRAPE_ANCESTORS` since we could have multiple leaves with the same parent.